installing/installing_azure_stack_hub/ipi/ipi-ash-preparing-to-install.adoc

```diff
@@ -0,0 +1,41 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ash-preparing-to-install-ipi"]
+= Preparing to install a cluster on Azure Stack Hub
+include::_attributes/common-attributes.adoc[]
+:context: ash-preparing-to-install-ipi
+
+toc::[]
+
+To prepare to install an {product-title} cluster on Azure Stack Hub, complete the following steps:
+
+* xref:../../../installing/overview/installing-preparing.adoc#installing-preparing[Select a cluster installation method].
+
+* xref:../../../installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc#installing-azure-stack-hub-account[Configure an Azure Stack Hub account] to host the cluster.
+
+* xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[Configure it to allow the sites] that your cluster requires access to.
+
+* Install Azure Stack Hub version 2008 or later.
+
+* Verify that you have approximately 16 GB of local disk space. Installing the cluster requires that you download the {op-system} virtual hard drive (VHD) cluster image and upload it to your Azure Stack Hub environment so that it is accessible during deployment. Decompressing the VHD files requires this amount of local disk space.
+
+In {product-title} version {product-version}, you can install a cluster on Microsoft Azure Stack Hub with an installer-provisioned infrastructure. However, you must manually configure the `install-config.yaml` file to specify values that are specific to Azure Stack Hub.
+
+[NOTE]
+====
+While you can select `azure` when using the installation program to deploy a cluster using installer-provisioned infrastructure, this option is only supported for the Azure Public Cloud.
+====
```

Comment:
I think these prerequisites are basically covered by the prerequisites in the [install assembly](https://80041--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/ipi/installing-azure-stack-hub-default#prerequisites_installing-azure-stack-hub-default-ipi) so they shouldn't be repeated here. Instead, this prerequisites section should list the steps that take place in the modules below, like we do [for AWS](https://docs.openshift.com/container-platform/4.16/installing/installing_aws/ipi/ipi-aws-preparing-to-install.html)